example/macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,632 @@
+// !$*UTF8*$!
```

Comment:
Check example folder and removed not necessary things.

Reply:
it is in example folder, is it?